sdk/purview/azure-purview-datamap/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+
+## 1.0.0b1 (2024-02-26)
```

Comment:
Please update date to latest before merge.

Reply:
Thanks. Updated